front/pages/api/w/[wId]/mcp/[serverId]/index.ts

```diff
@@ -124,9 +124,9 @@ async function handler(
         });
       }
 
-      const { name, url, description, tools } = req.body;
```

Comment:
ooc, i was  wondering while refactorign the form , do we really need to be able to update tools and url ? we don't do that so far ?

Reply:
i'm pretty sure we don't do that so far right, will remove it in this PR !